### PR TITLE
Issue 144

### PR DESCRIPTION
**Pull request by elma13**
In this branche a new approach with getting the data and storing the data is used. There are global reactiveValues in stead of the communication module. The communication module is still there for some calculations.
All visualisations, metatable and map are changed.

### CODE REVIEW

File: modules/add_show_availability.R [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+show_availability_server <- function(id,
+                                     data_to_show,
+                                     data_stations,
+                                     time_period
+) {
+
+  moduleServer(id, function(input, output, session) {
+
+    ns <- session$ns
```
Comment (Hendricxw) [inline modules/add_show_availability.R:33]: Can these rows be deleted?

**Review by Hendricxw — CHANGES_REQUESTED** [review]
I still get the warning:
Warning: Problem while computing `timestamp_table = as.POSIXct(...)`.
ℹ (0-based) yday 366 in year 2022 is invalid
However, this also gives some insight in how long it takes to get the data into the tool (multiple minutes...). Map still displays SSK_LH018 and no LTD_62781, but we will adress these issues in different issue :)

File: modules/add_show_availability.R (hunk 1 above)
Comment (elma13) [reply modules/add_show_availability.R:33]: Good point, I think they can indeed removed.

Comment (elma13) [reply modules/add_show_availability.R:33]: Yes those can be deleted

**Comment by elma13** [comment]
> I still get the warning: Warning: Problem while computing `timestamp_table = as.POSIXct(...)`. ℹ (0-based) yday 366 in year 2022 is invalid However, this also gives some insight in how long it takes to get the data into the tool (multiple minutes...). Map still displays SSK_LH018 and no LTD_62781, but we will adress these issues in different issue :)

Let's do this in a new issue. It is a warning, so it sort of all still works

**Pull request merged**